yamux/src/connection.rs

```diff
@@ -350,7 +353,7 @@ struct Active<T> {
     socket: Fuse<frame::Io<T>>,
     next_id: u32,
 
-    streams: IntMap<StreamId, Stream>,
+    streams: IntMap<StreamId, Arc<Mutex<Shared>>>,
```

Comment:
```suggestion
    streams: IntMap<StreamId, Arc<Mutex<stream::Shared>>>,
```

Adds more context to the name `Shared`, i.e. makes it clear that it is shared with the stream, and not e.g. with other connections.